src/components/modules/Authentcation/Login/Login.tsx

```diff
@@ -1,5 +1,153 @@
-import React from "react";
+import React, { useContext, useState } from "react";
+import { authuserdata, LoginForm } from "../../../../../interface/Interface";
+import { useForm } from "react-hook-form";
+import { Link, useNavigate } from "react-router-dom";
+import axios from "axios";
+import { AUTHURL } from "../../../../constans/EndPoints/ENDPOINT";
+import { toast } from "react-toastify";
+import { authcontext } from "../../../../Context/Authcontext";
+import { VALIDATIONS } from "../../../../constans/Validation/Validation";
 
 export default function Login() {
-  return <div>Login</div>;
+  const { saveuserdata } = useContext<authuserdata | null>(authcontext);
+  const [isloading, setislogin] = useState();
+  const navigateRegister = () => {
+    navigate("/Register");
+  };
+  const navigateLogin = () => {
+    navigate("/Login");
+  };
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<LoginForm>({ defaultValues: { email: "", password: "" } });
+  const navigate = useNavigate();
+  const onsubmit = async (data: LoginForm) => {
+    try {
+      const response = await axios.post(AUTHURL.LOGIN, data);
+      console.log(response);
+      toast.success(response.data?.data.message || "login is successfully");
+      localStorage.setItem("token", response.data.data.accessToken);
+      saveuserdata();
+      setislogin(response.data);
+      navigate("/dashboard");
+    } catch (error: any) {
+      toast.error(error);
+    }
+  };
+
+  return (
+    <>
+      <div className="py-10">
+        <div>
+          <h3 className="text-auttextcolor font-bold px-10">
+            Continue your learning journey with QuizWiz!
+          </h3>
+          <div className="p-10 w-full flex justify-start gap-8">
+            <div
+              className="bg-authchoosuserbackgroundcolor w-48 text-center py-5 rounded-xl transition-all  cursor-pointer text-auttextcolor hover:outline outline-auttextcolor"
+              onClick={navigateLogin}

```

Comment:
Why not making this as  a link and give it the styles? navigate function should only be used for programmatic navigation which means that we have other logic that we need to perform beside the navigation. Otherwise, we should be using the Link component.